src/Configuration/CFGCommon/CFGCommon.cpp

```diff
@@ -110,6 +115,14 @@ void CFG_post_msg(const std::string& message) {
   }
 }
 
+void CFG_post_warning(const std::string& message) {
+  if (m_msg_function != nullptr) {
+    m_msg_function(message);
+  } else {
+    printf("Warn : %s\n", message.c_str());
```

Comment:
Why not use the python logging module here ?

Reply:
Hi Ravi, can you elaborate more on python logging module? Maybe point to an example. Is it similar to FOEDAG::Compiler::Message/ErrorMessage()? Or there is other printing msg API?

Reply:
Hi Ravi, can you elaborate more on python logging module? Maybe point to an example, thanks.
Is it similar to FOEDAG::Compiler::Message/ErrorMessage()? Or there is other printing msg API? 

BTW, this is c++ code, I am not sure how it is related to Python:
a. maybe we really support Python module in c++ which I am not aware of
b. maybe you mistakenly think this is .py file

Reply:
Had a convo, we don't need to use compiler logging module here or in python code.